app/models/resource.py

```diff
@@ -0,0 +1,15 @@
+from .. import db
+
+
+class Resource(db.Model):
+    __tablename__ = 'resources'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64))
+    address = db.relationship('Address', backref='resource', lazy='joined',
```

Comment:
Let's delete this line, then the pr can stand on it's own and it's changed in https://github.com/hack4impact/women-veterans-rock/pull/7 anyway.